//package ch09;
//
//public class TryCatch3Demo {
//    public static void main(String[] args) {
//        int[] array = {0, 1, 2};
//        try {
//            int x = array[3];
//        } catch (Exception e) {
//            System.out.println("아구");
//        } catch (ArrayIndexOutOfBoundsException e) {
//            System.out.println("존재 x");
//        }
//        System.out.println("끝");
//    }
//}
